src/components/App/App.tsx

```diff
@@ -1331,15 +1360,19 @@ export default class App extends React.Component<null, AppState> {
                       )}
                     </div>
                     {this.state.fullScreen && (
-                      <Chat
-                        className="fullScreenChat"
-                        chat={this.state.chat}
-                        nameMap={this.state.nameMap}
-                        pictureMap={this.state.pictureMap}
-                        socket={this.socket}
-                        scrollTimestamp={this.state.scrollTimestamp}
-                        getMediaDisplayName={this.getMediaDisplayName}
-                      />
+                      <div className={classes.Chat}>
+                        <Chat
+                          className="fullScreenChat"
+                          chat={this.state.chat}
+                          nameMap={this.state.nameMap}
+                          onUserNameChange={this.updateName}
```

Comment:
What's the purpose of the additional props? I assume it's currently able to do what it needs with the socket passed to it